db/migrate/20220308121052_create_recipes.rb

```diff
@@ -0,0 +1,14 @@
+class CreateRecipes < ActiveRecord::Migration[7.0]
+  def change
+    create_table :recipes do |t|
+      t.string :name
+      t.float :preparation_time
+      t.float :cooking_time
```

Comment:
type attribut, should be time here for preparation_time and cooking_time

Reply:
That is not a time @Taher-web-dev it is a duration :relieved:
As you can guess that you can cook at any time :sweat_smile:
please read the instructions before giving any review.

Reply:
OK , you have right. happy to see you understand this instruction.